### DISCUSSION TITLE
Multiple checklists for different situations

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-checklists#447] [Feature Requests]
> Can you create multiple content checklists and set them to appear based on certain filters?
> 
> Here's a use case. It's likely that users may have different checklists based on the type of content. For example, I want the author/editor to check image size, SEO and grammar for blog posts but do something different for video posts.
> 
> So my question is can I have multiple content checklist and have checklist #1 show for Blog Posts, checklist #2 show for Video Posts (or a different custom post type)?

**Comment by andergmartins** [comment]
Right, we can follow a similar approach as we used for the notification workflows. We can remove from the settings page, and create a new post type for checklists, displaying filters and fields to customize the warnings, settings and other behaviours. This can help on adding room to make it more flexible, allowing to work better combined with 3rd party plugins like Yoast SEO, Woocommerce, etc
